experiments/AMIP/moist_mpi_earth/bucket/bucket_init.jl

```diff
@@ -0,0 +1,167 @@
+# slab_rhs!
+using ClimaCore
+using CLIMAParameters: AbstractEarthParameterSet
+using CLIMAParameters.Planet: ρ_cloud_liq
+using ClimaLSM
+using ClimaLSM.Bucket: BucketModel, BucketModelParameters, AbstractAtmosphericDrivers, AbstractRadiativeDrivers
+
+import ClimaLSM.Bucket: surface_fluxes, surface_air_density, liquid_precipitation, BulkAlbedo, surface_albedo
+
+using ClimaLSM: make_ode_function, initialize_prognostic, initialize_auxiliary
+
+import ClimaLSM: initialize
+
+struct EarthParameterSet <: AbstractEarthParameterSet end
+
+struct BucketSimulation{P, Y, D, I}
+    params::P
+    Y_init::Y
+    domain::D
+    integrator::I
+end
+
+# init simulation
+function initialize(model::BucketModel, space::ClimaCore.Spaces.AbstractSpace)
```

Comment:
Does it make sense to add this method to ClimaLSM @kmdeck? Probably something we will use again. We can merge this PR like this and open a PR for LSM, order doesn't really matter.

Reply:
agreed! I'd say let's wait on moving this to LSM until we have another land model we want to couple & the exact form of the Simulation struct is finalized. perhaps we will have a LandSimulation with an addition field = model?

Reply:
i misread this! the #init simulation comment threw me off. yeah let's move this into ClimaLSM